src/app/modules/auth/login.page.tsx

```diff
@@ -1,15 +1,18 @@
+import Button from '@atomic/atm.button/button.component';
+
 function LoginPage() {
  return (
-  <div className="flex flex-col items-center bg-amber-100 min-h-screen w-full">
+  <div className="flex flex-col items-center bg-brand-accessory-x-light min-h-screen w-full">
    <header className="flex justify-center items-center h-30">
-    <h1 className="text-4xl font-serif text-purple-300">Login</h1>
+    <h1 className="text-4xl font-serif text-grayScale-dark text-xx-large p-medium">Login</h1>
    </header>
    <div className="h-64 w-56 bg-white rounded-2xl text-center flex flex-col justify-center items-center">
     <form action="" className="flex flex-col justify-center items-center">
      <label htmlFor="email">email</label>
      <input type="text" className="generic-input" name="email" id="" />
      <label htmlFor="password">password</label>
      <input type="text" className="generic-input" name="password" id="" />
+     <Button>Enviar</Button>
```

Comment:
Geralmente nos projetos do front extraímos as strings em pt-br para um arquivo a parte para isolar as responsabilidades, aqui por exemplo poderia criar um arquivo no mesmo diretório chamado:
`login.page.strings.ts` com um objeto assim:
```suggestion
   export const loginPageStrings = {
   sumbmitButton: "Enviar"
   labels: {
     email: "e-maill"
     password: "senha"
   }
   }
```

Reply:
Ai vc importa esse objeto aqui e usa essas strings no lugar de deixar o texto que ficará disponível para o usuário juntamente do código, isso ajudar a dar manutenção conforme for crescendo